src/api/app/views/webui/users/index.html.haml

```diff
@@ -18,9 +18,14 @@
       %tbody
 
     .pt-4
-      = link_to(new_user_path(pagetitle: 'create user', submit_btn_text: 'create')) do
-        %i.fas.fa-plus-circle.text-primary
-        Create User
+      - if feature_enabled?(:responsive_ux)
+        - content_for :actions do
+          %li.nav-item
+            = link_to_action('Create User', new_user_path(pagetitle: 'create user', submit_btn_text: 'create'), icons: 'fa-plus-circle fa-lg mr-2')
```

Comment:
@danidoni, maybe you could also move the other classes inside the helper `fa-lg mr-2`.

Reply:
I think it would easier for future changes and to have all the icons with the same size and margin. The same happens in your other PR https://github.com/openSUSE/open-build-service/pull/10263, these classes are repeated.

Reply:
You are right, again!

Reply:
I've decided to not use a helper method, so this won't be needed.